homeassistant/components/holiday/config_flow.py

```diff
@@ -128,7 +161,11 @@ async def async_step_options(
             data = {CONF_COUNTRY: country}
             options: dict[str, Any] | None = None
             if province := user_input.get(CONF_PROVINCE):
-                data[CONF_PROVINCE] = province
+                # Don't store the user alias but the stable ISO 3166 code
+                province_code = await self.hass.async_add_executor_job(
```

Comment:
Is says `province_code` but I feel pretty sure this is returning and alias and not the code?

Reply:
Got rid of it all.